ui/raidboss/data/06-ew/raid/p2n.ts

```diff
@@ -132,13 +132,7 @@ const triggerSet: TriggerSet<Data> = {
       netRegexDe: NetRegexes.startsUsing({ id: '67F6', source: 'Hippokampos', capture: false }),
       netRegexFr: NetRegexes.startsUsing({ id: '67F6', source: 'Hippokampos', capture: false }),
       netRegexJa: NetRegexes.startsUsing({ id: '67F6', source: 'ヒッポカムポス', capture: false }),
-      alertText: (_data, _matches, output) => output.text!(),
-      outputStrings: {
-        text: {
-          en: 'Aoe--Get on grid',
-          de: 'AoE--Geh auf die Gitter',
-        },
-      },
+      response: Responses.aoe(),
```

Comment:
I think the "get on grid" part is a little bit much.  There's a ground aoe many seconds later that indicates that you should stand on the grid (or the corners that aren't grids), and so I think just saying "aoe" is enough here.